### PR TITLE
Release 1.6.1-gcm.0

### PR DESCRIPTION
**Pull request by maelvls**
I did not release 1.6.0-gcm.0 since 1.6.1 was released the same day I did the GCM update. So I went directly to 1.6.1.

- Draft release: ~https://github.com/jetstack/jetstack-secure-gcm/releases/tag/untagged-46cf5afbfd97cce01433~
- Final release: https://github.com/jetstack/jetstack-secure-gcm/releases/tag/1.6.1-gcm.0

### CODE REVIEW

**Comment by maelvls** [comment]
1.4.4-gcm.0, 1.5.4-gcm.1 and 1.6.1-gcm.0 have been accepted!

@wallrj Would you like to review this PR?

File: chart/jetstack-secure-gcm/charts/cert-manager/Chart.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 annotations:
   artifacthub.io/prerelease: "false"
 apiVersion: v1
-appVersion: v1.5.3
+appVersion: v1.6.0
```
Comment (wallrj) [inline chart/jetstack-secure-gcm/charts/cert-manager/Chart.yaml:4]: In the PR description you talk about jumping straight to 1.6.1 so this and other references to 1.6.0 need updating, right?

**Review by wallrj — CHANGES_REQUESTED** [review]
Sorry for the delay in reviewing this.
There are some 1.6.0 references still in here,
and the "Draft release" link in the PR description no longer works for me.

File: .gitignore [hunk 2]
```diff
@@ -1,3 +1,4 @@
 ca.crt
 ca.key
 *.t.err
+*~
```
[the remark this answers is not in the document]
Comment (maelvls) [reply .gitignore:4]: That's because of `sed -i~` (only way to make sed compatible on both macOS and Linux). But it seems like I do not use `sed -i~` anywhere, so this line can be removed.

File: chart/jetstack-secure-gcm/charts/cert-manager/Chart.yaml (hunk 1 above)
Comment (maelvls) [reply chart/jetstack-secure-gcm/charts/cert-manager/Chart.yaml:4]: I think I got confused somehow, or I didn't update the PR description to say "this PR is for 1.6.0-gcm.0". I need to investigate what I did.

Looking at the step-by-step recording in https://github.com/jetstack/jetstack-secure-gcm/releases/tag/1.6.1-gcm.0, the 1.6.1-gcm.0 that I release on the Google Marketplace is weird:

- I used the cert-manager git tag `v1.6.1` which seems OK,
- I used the Helm chart 1.6.0 (the chart did not change so that's not a huge deal, but it is still weird),
- I think I messed up with `sed -i~ "s/publishedVersion: .*/publishedVersion: $APP_VERSION/" schema.yaml`. I must have run it once with 1.6.0 and forgot to re-run it for 1.6.1.

Weirdly enough, I also seem to have pushed the images (but not published on the UI) the version `1.6.0-gcm.0`.

Let's check which `publishedVersion` has been set. For 1.6.1-gcm.0:

```sh
$ docker run -v $PWD:/opt/mount --rm --entrypoint=cat gcr.io/jetstack-public/jetstack-secure-for-cert-manager/deployer:1.6.1-gcm.0 /data/schema.yaml | grep -i publishedVersion:
  publishedVersion: 1.6.1-gcm.0
```

For 1.6.0-gcm.0:

```sh
$ docker run -v $PWD:/opt/mount --rm --entrypoint=cat gcr.io/jetstack-public/jetstack-secure-for-cert-manager/deployer:1.6.0-gcm.0 /data/schema.yaml | grep -i publishedVersion:
  publishedVersion: 1.6.0-gcm.0
```

To summarize, I did two releases (1.6.0-gcm.0 and 1.6.1-gcm.0) with the correct cert-manager and publishedVersion (except for the Helm chart that stayed at 1.6.0, but no big deal), but then I only published one on the GCM UI and on GitHub Releases.
